packages/components/src/SVG/icons/EmailIcon.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import { SVG, SVGProps } from "~components/SVG"
+
+export const EmailIcon = (props: Omit<SVGProps, "children">): JSX.Element => (
```

Comment:
I followed the guide but the icons don't render. May be to do with the component they are used in. Didn't have time to investigate. Any immediate ideas?

Reply:
Ah you're right, we have to make a compatible copy of BrandMoment. I'll hop in and help you with that.